kalite/updates/api_views.py

```diff
@@ -17,31 +17,20 @@
 from django.utils import translation
 from django.utils.translation import ugettext as _
 
-from . import delete_downloaded_files, get_local_video_size, get_remote_video_size
-from .models import UpdateProgressLog, VideoFile
+from .videos import delete_downloaded_files
+from .models import UpdateProgressLog
 from .views import get_installed_language_packs
+from .download_track import VideoQueue
 from fle_utils.chronograph.utils import force_job
 from fle_utils.django_utils.command import call_command_async
 from fle_utils.general import isnumeric, break_into_chunks, softload_json
 from fle_utils.internet.decorators import api_handle_error_with_json
 from fle_utils.internet.classes import JsonResponse, JsonResponseMessageError, JsonResponseMessageSuccess
 from fle_utils.orderedset import OrderedSet
-from kalite.i18n import get_youtube_id, get_video_language, lcode_to_ietf, delete_language, get_language_name
+from kalite.i18n.base import lcode_to_ietf, delete_language, get_language_name
 from kalite.shared.decorators.auth import require_admin
-from kalite.topic_tools.settings import TOPICS_FILEPATHS, CHANNEL
-
-
-
-def divide_videos_by_language(youtube_ids):
```

Comment:
No longer required - we now only allow adding to download queue per language (this was actually de facto the only way to add stuff due to the front end interface, but our API supported a more liberal way).